sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -616,4 +631,4 @@ def nodes_dest_ip(self) -> Optional[str]:
 
 
 class MonitorSetMinikube(MonitorSetGCE):
-    def install_scylla_manager(self, node, auth_token): ...
+    def install_scylla_manager(self, node): ...
```

Comment:
Is it pylint fix? Anyway, it doesn't look related to the goal of the commit.

Reply:
yes, it is pylint issue